models/mobilenetv1.py

```diff
@@ -111,13 +111,21 @@ class MobileNetV1(Model):
 
     Args:
         scale (float): scale of channels in each layer. Default: 1.0.
-        class_dim (int): output dim of last fc layer. Default: 1000.
+        num_classes (int): output dim of last fc layer. Default: -1.
```

Comment:
need more clearly.   -1 means not  include the fully-connected layer at the top/output of the network.

Reply:
done, thanks.